### PR TITLE
Coin icons fixed

### PR DESCRIPTION
**Pull request by HERRKIN**

### CODE REVIEW

File: obi-wallet/apps/mobile/src/app/screens/components/coin-icon/index.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Image, StyleSheet, View } from "react-native";
+import { SvgProps } from "react-native-svg";
+import SvgImage from "react-native-svg/lib/typescript/elements/Image";
```
Comment (inyono) [inline obi-wallet/apps/mobile/src/app/screens/components/coin-icon/index.tsx:4]: That import looks strange.

File: obi-wallet/apps/mobile/src/app/balances/index.ts [hunk 2]
```diff
@@ -39,6 +40,7 @@ export function formatCoin(coin: ExtendedCoin) {
       const digits = 6;
       const usdValue = coin.usdPrice / Math.pow(10, digits);
       const amount = parseInt(coin.amount, 10) / Math.pow(10, digits);
+      console.log({ denom });
```
Comment (inyono) [inline obi-wallet/apps/mobile/src/app/balances/index.ts:43]: console.log

**Review by inyono — APPROVED** [review]

File: obi-wallet/apps/mobile/src/app/screens/components/coin-icon/index.tsx (hunk 1 above)
Comment (HERRKIN) [reply obi-wallet/apps/mobile/src/app/screens/components/coin-icon/index.tsx:4]: I think it was an autoimport I forgot to delete. I will do in my next pr

File: obi-wallet/apps/mobile/src/app/balances/index.ts (hunk 2 above)
Comment (HERRKIN) [reply obi-wallet/apps/mobile/src/app/balances/index.ts:43]: will do

**Pull request merged**